core/templates/components/question-difficulty-selector/question-difficulty-selector.component.html

```diff
@@ -27,6 +27,14 @@
     top: 1.1rem;
   }
 
+  .mat-radio-button /deep/ .mat-radio-label {
+    align-items: flex-start;
+  }
+
+  .mat-radio-button /deep/ .mat-radio-label .mat-list-item {
+    height: auto;
+  }
```

Comment:
Is there some way we avoid usin `/deep/`? It looks like this is deprecated and may be removed soon (https://angular.io/guide/component-styles#deprecated-deep--and-ng-deep).

Reply:
> Is there some way we avoid usin `/deep/`? It looks like this is deprecated and may be removed soon (https://angular.io/guide/component-styles#deprecated-deep--and-ng-deep).

It is possible to achieve this if we put the css in the global css file like in opia-material.css or oppia.css. Will do this.

Reply:
Made the appropriate changes @kevintab95 . PTAL !

Reply:
@kevintab95 PTAL. Thanks!